superdesk/metadata/item.py

```diff
@@ -308,6 +308,16 @@
         'type': 'string'
     },
 
+    # Desk and Stage Details
```

Comment:
If ingest also needs these properties then it's better to move from common to metadata/item